packages/nodejs/src/client.ts

```diff
@@ -159,6 +170,52 @@ export class BaseClient implements Client {
     return demo(this.tracer())
   }
 
+  /**
+   * Sets up the AppSignal logger with the output based on the `log` config option. If
+   * the log file is not accessible, stdout will be the output.
+   */
+  private setUpLogger(): winston.Logger {
+    const logFilePath = this.config.logFilePath
+    const logLevel = String(this.config.data["logLevel"])
+    let transport
+
+    const logFormat = printf(({ level, message, timestamp }) => {
+      return `[${timestamp} (process) #${process.pid}][appsignal][${level}] ${message}`
+    })
+
+    if ((this.config.data["log"] = "file" && logFilePath)) {
```

Comment:
```suggestion
    if (this.config.data["log"] = "file" && logFilePath) {
```

Reply:
I also forgot an equal sign 🤦🏼 . I really need to find a way of testing this when I write the logger class.